torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1728,8 +1728,6 @@ Tensor binary_cross_entropy_double_backward_grad_output(const Tensor & grad, con
   }
   if (reduction == at::Reduction::Mean) {
     return ggO / input.numel();
-  } else if (reduction == at::Reduction::Sum) {
-    return ggO.sum();
   }
```

Comment:
This is a clear bug, as shape of `gg0` has to be same as shape of `self`. I do not know why `gradgradcheck` missed that, CC @albanD .